lib/vmpooler/pool_manager.rb

```diff
@@ -156,8 +156,14 @@ def _check_ready_vm(vm, pool_name, ttl, provider)
         $redis.hset('vmpooler__vm__' + vm, 'check', Time.now)
         # Check if the hosts TTL has expired
         if ttl > 0
-          # host['boottime'] may be nil if host is not powered on
-          if ((Time.now - host['boottime']) / 60).to_s[/^\d+\.\d{1}/].to_f > ttl
+          # 'boottime' may be nil if host is not powered on
```

Comment:
Thanks for your contribution @quorten and for squishing this bug! Could we update the comment to be more relevant?
```suggestion
          # if 'boottime' is nil, set bootime to beginning of unix epoch, forces TTL to be assumed expired
```

Reply:
Sure thing, looks good.